backend/app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+@posts = Post.all(:select => "title, author, id, content, posted_at", :order => "posted_at DESC", :limit => 20) 
```

Comment:
What are posts? We don't have a `Post` model. Our article model is called `Article`.

Also this doesn't seem like a valid controller code. Where's the controller class. See `articles_controller.rb` for an example and there is probably where you want to add the `respond_to` block to respond to both json and rss.